packages/codesandbox-theme-docs/src/misc/theme.tsx

```diff
@@ -197,116 +197,118 @@ const H6 =
       <HeaderLink tag="h6" context={context} {...props}>
         {children}
       </HeaderLink>
-    )
-  }
+    );
+  };
 
 const A = ({
   children,
+  href,
   ...props
 }: {
-  children?: React.ReactNode
-  href?: string
+  children?: React.ReactNode;
+  href?: string;
 }) => {
-  const isExternal = props.href && props.href.startsWith('https://')
+  const isExternal = href && href.startsWith("https://");
   if (isExternal) {
     return (
       <a target="_blank" rel="noreferrer" {...props}>
         {children}
       </a>
-    )
+    );
   }
-  return props.href ? (
-    <Link href={props.href}>
+
+  return href ? (
+    <Link href={href} passHref>
       <a {...props}>{children}</a>
     </Link>
   ) : (
     <></>
-  )
-}
+  );
+};
```

Comment:
The actual change. I've deconstructed `href` and instead of passing `...props` including `href` use the `Link` to `passHref`. Next adds the `basePath` to the link, but by using the href prop before we overrode it without the base path!